Multisig/Features/Connect to Web/UI/Send Transaction Request/SendTransactionRequestViewController.swift

```diff
@@ -581,7 +581,7 @@ class SendTransactionRequestViewController: WebConnectionContainerViewController
         App.shared.snackbar.show(error: gsError)
 
         Tracker.trackEvent(.executeFailure, parameters: [
-            "chainId": self.chain.id!
+            "chain_id": self.chain.id!
```

Comment:
Is `self.chain.id!` the same that is used for `shouldAddChainIdParam()`?

Reply:
hm... in the case of this screen I would use the `self.chain.id` because I expect this to be moved to some place in the future and the tracking will become chain-specific for a selected chain. For example, when creating safe, the selected chain could be different from the chain of selected safe. I'd keep it like it is here.